locale/en.json

```diff
@@ -403,6 +404,16 @@
           "description": "A Tribunal Caseworker is looking at your answers and will contact you to tell you what to do next.",
           "dueDate": "This should be by <b>{{ applicationNextStep.deadline }}</b> but it might take longer than that."
         },
+        "ended": {
+          "description": "ended desc",
+          "dueDate": "ended  should be by <b>{{ applicationNextStep.deadline }}</b> but it might take longer than that.",
+          "ctaTitle": "Your appeal has now ended",
+          "ctaInstruction": "Review your <a href=\"{{ paths.common.detailsViewers.appealDetails }}\">Notice of Ended Appeal</a>. This includes details of who ended the appeal and why.",
+          "ctaReview": "If a Tribunal Caseworker ended the appeal and you disagree with this decision, you have 14 days to ask for the decision to be reviewed by a judge.",
+          "ctaContact": "You can do this by emailing <a href=\"mailto:{{ applicationNextStep.hearingCentreEmail }}\">{{ applicationNextStep.hearingCentreEmail }}</a>. Please include your Appeal reference in the subject line of the email.",
```

Comment:
@gavinslilley I think we need to differentiate the hearingCentre before and after listing and the email (designated hearing centre) could change based on that. From what I see only the hearingCentre value is used and did not find changes to distinguish the two. Could you please re-check once?